app/ios/Podfile

```diff
@@ -11,7 +11,7 @@ require Pod::Executable.execute_command("node", ["-p",
 
 project "Self.xcodeproj"
 
-platform :ios, "15.0" if !ENV["ACT"]
+platform :ios, "15.1" if !ENV["ACT"]
```

Comment:
even though we aren't using credentials manager for ios, it breaks the build if we use `15.0`